src/frontend/src/flows/addDevice/manage/addDevicePickAlias.ts

```diff
@@ -51,4 +53,26 @@ const init = (): Promise<string | null> =>
       event.preventDefault();
       resolve(deviceAlias.value);
     };
+    const deviceInput = document.getElementById(
+      "deviceAlias"
+    ) as HTMLInputElement;
+
+    deviceInput.addEventListener("invalid", () => {
+      const firstChar = deviceInput.value[0];
+      const lastChar = deviceInput.value.slice(-1);
```

Comment:
Do you think it'd be clearer to check with `startsWith` and `endsWith`? or more confusing?